networks/_modules/ignite/main.tf

```diff
@@ -1,3 +1,12 @@
+terraform {
+  required_providers {
+    quorum = {
+      source = "trung/quorum"
```

Comment:
to remove?

Reply:
changed to `ConsenSysQuorum/quorum` as now we source the provider from our "temporary" official [Terraform Registry](https://registry.terraform.io/providers/ConsenSysQuorum/quorum/latest)